install/install.sh

```diff
@@ -40,6 +40,7 @@ spack install --fail-fast --no-check-signature
 # Install parsl and black
```

Comment:
Need to add `isort` to this comment if all runs smoothly. I decided to leave `black` and `isort` closely coupled together in all coding blocks since they are both under the category of a `Format Check`.